easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-GCC-11.3.0.eb

```diff
@@ -24,11 +26,6 @@ toolchainopts = {'pic': True}
 
 default_easyblock = 'ConfigureMake'
 
-dependencies = [
-    ('GDGraph', '1.56'),
-    ('Perl', '5.34.1'),
```

Comment:
@deniskristak It looks like you're accidentally changing an existing easyconfig here?